tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "expo/tsconfig.base",
   "compilerOptions": {
```

Comment:
required for dynamic imports in `clients`. Will be used to avoid needlessly mocking unused transitive dependencies